recipes/ipython-memory-usage/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipython-memory-usage" %}
+{% set version = "1.1" %}
+{% set sha256 = "1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ianozsvald/ipython_memory_usage.git
```

Comment:
Please build from a tarball rather than a repo:

http://conda-forge.readthedocs.io/en/latest/meta.html#build-from-tarballs-not-repos

You should be able to pull the release from pypi.io and use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18